cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/constants/AnomalyDetectorConfig.java

```diff
@@ -227,6 +228,31 @@ private AnomalyDetectorConfig() {
   public static final String DEFAULT_MAINTENANCE_EVENT_CLASS = MaintenanceEvent.class.getName();
```

Comment:
Is it reasonable to think about creating another static class `MaintenanceEventConfig` for all configs related to maintenance event?

Reply:
`MaintenanceEvent` is an anomaly -- it extends from it -- and all anomaly configs are located in `AnomalyDetectorConfig`.